### ISSUE TITLE
Add payment succeeded webhook handler.

### ISSUE THREAD

**Issue by thorsten-stripe** [ynnoj/stripe-paymentintents-example#10]
For integrations using the automatic confirmation flow a webhook handler is needed to process the `PaymentIntent.succeeded` in case the connection between the frontend and server drops: https://stripe.com/docs/payments/payment-intents/quickstart#fulfillment

**Comment by ynnoj** [comment]
@thorsten-stripe Thanks for the tip! My understanding that this is only really a requirement when fulfilling an order? Is that required in this example when there's no concept of an order to fulfil?

Happy to provide an example handler for the web hook, just not sure how that would fit into this example in that scenario.

**Comment by thorsten-stripe** [comment]
Yes, fair, we just don't want folks to accidentally forget about it. Maybe you can implement the frontend success handler[0] and put a comment in there along the lines of: "We're using the PaymentIntent status returned by Stripe.js here to inform our customer about the outcome of the payment. Should we need to update our backend system with the payment status we enable a webhook handler as outlined here: https://stripe.com/docs/payments/payment-intents/quickstart#fulfillment". Wdyt?

[0] https://github.com/ynnoj/stripe-paymentintents-example/issues/4

**Comment by ynnoj** [comment]
@thorsten-stripe Sounds good! I'll hook up a handler from the Stripe event.
